<!DOCTYPE HTML>
<html>
	<head>
		<title>Project 3</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
        <style>
            /* Add a black box style */
            .black-box {
            background-color: rgba(255, 255, 255, 0.055);
            padding: 20px; /* Add padding for spacing */
            color: white; /* Set text color to white */
            margin: 0 auto;
            width: 100vw;
            height: 100vh
        }

            .navigation {
			display: flex;
			justify-content: space-between;
			margin-top: 10px;
		}

		    .navigation a {
			padding: 5px 10px;
			background-color: #131313;
			color: white;
			text-decoration: none;
			border-radius: 3px;
		}

            .embed-container {
            padding-top: 50px; /* Create space at the top */
            width: 100%;
            height: 100%;
            background-color: white; /* Background color for the space at the top */
        }
		
        </style>

	</head>

	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

                    <article>
                        <div class="black-box">
                            <h1>Automated Workflow for Retrieving Unannotated RNA Sequences Using R </h1>
                            <p>Transcripts often contain regulatory elements, coding sequences, and other biologically significant motifs that are critical for understanding gene function and expression. While RNA sequences can usually be obtained easily using tools like the **biomaRt** package in R, retrieving less common regions such as sequences not normally expressed in transcriptsâ€”requires a different approach. Efficient extraction and processing of these sequences are essential for a wide range of downstream analyses, including motif discovery.</p>
                            <p>This workflow is designed to address this challenge by using a combination of bioinformatics packages in R. It automates the retrieval and processing of unannotated RNA sequences by handling chromosomal locations, strand orientation, and sequence manipulation (e.g., reverse complementing sequences on the negative strand). This method replaces labor-intensive manual processes with an efficient and reproducible solution for specialized sequence analysis. </p>                            <p>For this project, I have also generated an R notebook embedded below. If you have any suggestion to improve the analysis or interested to collaborate, please feel free to contact me :D </p>
                            <br>
                            <div class="embed-container">    
                                <iframe src="exon seq notebook v2.nb.html" width=100% height=100% frameborder="0"></iframe>
                            </div>
                                <nav class="nav_top bookmark-tabs">
                                    <ul>
                                        <li><a href="index.html#intro">Intro</a></li>
                                        <li><a href="index.html#Project">Project</a></li>
                                        <li><a href="index.html#Publication">Publication</a></li>
                                        <li><a href="index.html#ScientificIllustration">Sci-Illust</a></li>
                                        <li><a href="index.html#contact">Contact</a></li>
                                    </ul>
                                </nav>
                                <div class="navigation">
                                    <a href="Project3.html">Previous project</a>
                                    <a href="index.html#Project" >Back to project page</a>
                                </div>
                        </div>
                    </article>
            </div>

		<!-- Footer -->
         <footer id="footer">
        <p class="copyright">&copy; Lee Jin</p>
        </footer>

        <!-- BG -->
        <div id="bg"></div>

        <!-- Scripts -->
        <script src="assets/js/jquery.min.js"></script>
        <script src="assets/js/browser.min.js"></script>
        <script src="assets/js/breakpoints.min.js"></script>
        <script src="assets/js/util.js"></script>
        <script src="assets/js/main.js"></script>
        <script>
            function loadSection(sectionUrl) {
                window.location.hash = 'Project/' + sectionUrl; // Update the URL hash
                loadContent(sectionUrl); // Load the content dynamically
            }
            
            function loadContent(sectionUrl) {
                var xhr = new XMLHttpRequest();
                xhr.open('GET', sectionUrl, true);
                xhr.onreadystatechange = function() {
                    if (xhr.readyState === 4 && xhr.status === 200) {
                        document.getElementById('sectionContent').innerHTML = xhr.responseText;
                    }
                };
                xhr.send();
            }
        </script>

    </body>
</html>
